src/main/java/com/msclub/meetingmanager/model/microsoft/MicrosoftTeamsMeetingDetails.java

```diff
@@ -16,6 +16,15 @@
     public boolean allowNewTimeProposals;
     public boolean isOnlineMeeting;
     public String onlineMeetingProvider;
+    public String title;
```

Comment:
this is not the change u should do. create a class similar to MSTeamsInterviewDetails. name it as MSTeamsInternalMeetingDetails.

meetingName,startDateTime,endDateTime,emailList as attributes and set setters getters constructors similar in MSTeamsInterviewDetails. 

Then in controller do this change
    @PostMapping("/internalmeeting/schedule")
    public String scheduleInternalMeeting(@RequestBody  MSTeamsInternalMeetingDetails msTeamsInternalMeetingDetails){
        return microsoftTeamsService.scheduleMicrosoftMeeting(msTeamsInternalMeetingDetails, MicrosoftTeamsMeetingType.INTERNAL_MEETING);
    }

then in service
      case INTERNAL_MEETING:
                             microsoftTeamsMeetingDetails.setSubject(MSTeamsInternalMeetingDetails.getTitle());